tests/unit-tests/gconstruct/test_gconstruct_utils.py

```diff
@@ -467,15 +467,37 @@ def test_read_index():
     df.to_parquet('/tmp/test_idx.parquet')
     split_info = {"train": "/tmp/test_idx.parquet", "column": ["ID"]}
     parquet_content, _, _ = read_index(split_info)
-    np.array_equal(parquet_content, data)
+    assert np.array_equal(parquet_content, data)
+
+    data_multi = ["p702", "p712", "p722", "p732", "p742", "p752"]
+    df = pd.DataFrame(data_multi, columns=['ID'])
+    # test with wildcard
+    df.to_parquet('/tmp/test_idx_multi.parquet')
```

Comment:
Can you test two files in the wildcard test?

Reply:
Check here:  ` split_info = {"train": "/tmp/test_idx*.parquet", "column": ["ID"]}`

Reply:
In this wildcard test we already had '/tmp/test_idx.parquet' above, we just need to create one more new file